it/java/gh/src/test/java/GHAPITest.java

```diff
@@ -0,0 +1,23 @@
+import apisdk.ApiClient;
+import com.microsoft.kiota.ApiException;
+import com.microsoft.kiota.authentication.AnonymousAuthenticationProvider;
+import com.microsoft.kiota.http.OkHttpRequestAdapter;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class BasicAPITest {
+
+    @Test
+    void basicTest() throws Exception {
+        var adapter = new OkHttpRequestAdapter(new AnonymousAuthenticationProvider());
+        adapter.setBaseUrl("http://127.0.0.1:1080");
+        var client = new ApiClient(adapter);
+
+        client.repos().byOrgId("my-owner").byRepoId("my-repo").get();
+        client.repos().byOrgId("my-owner").byRepoId("my-repo").generate().post(null);
```

Comment:
If I forget about the internals of Kiota I truly cannot explain this as it doesn't map to the original description.